mock-identity-system/src/main/resources/messages.properties

```diff
@@ -8,7 +8,9 @@ invalid_phone=Invalid Phone number
 invalid_encoded_photo=Invalid encoded photo
 invalid_postalcode=Invalid postal code
 invalid_date_of_birth=Invalid Date of birth
-invalid_fullname=Invalid fullName
+invalid_fullname=Invalid Full Name
+invalid_firstname=Invalid First Name
+invalid_lastname=Invalid Last Name
 invalid_gender=Invalid gender
 invalid_street_address=Invalid street address
 invalid_locality=Invalid Locality
```

Comment:
I am not able to find the code change in 
https://github.com/mosip/esignet-mock-services/blob/develop/mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/AuthenticationServiceImpl.java#L262

where we should send the first name and last name if consented by the user ?

Reply:
We should also support all the standard claims here